cpp/ql/lib/semmle/code/cpp/ir/implementation/aliased_ssa/Operand.qll

```diff
@@ -269,6 +271,10 @@ class RegisterOperand extends NonPhiOperand, TRegisterOperand {
 
   final override string toString() { result = tag.toString() }
 
+  // most RegisterOperands have a more meaningful location at the definition
+  // the only exception is ThisArgumentOperand
```

Comment:
```suggestion
  // most `RegisterOperands` have a more meaningful location at the definition. The only exception is `ThisArgumentOperand`.
```

Reply:
by the way, do we have a conventional maximum line width? Is it something that the codeql autoformatter will be able to fix on its own?

Reply:
We don't really have one, no. I think it was a mistake from my side to squash these two down to a single line. I really just wanted to fix the missing quotes.